src/dodal/beamlines/i05.py

```diff
@@ -17,6 +25,39 @@ def synchrotron() -> Synchrotron:
     return Synchrotron()
 
 
+# BL05 shared devices
+
+
+@device_factory()
+def pgm_i05() -> PGM:
+    return pgm()
+
+
+@device_factory()
+def m1() -> XYZPitchYawRollStage:
+    return m1_collimating_mirror()
+
+
+# will connect after https://jira.diamond.ac.uk/browse/I05-731
+@device_factory()
+def m3mj6() -> XYZPiezoSwitchingMirror:
+    return m3mj6_switching_mirror()
+
+
+# beamline specific devices
+
+
+# will connect after https://jira.diamond.ac.uk/browse/I05-731
+class M4M5Mirror(StrictEnum):
+    UNKNOWN = "Unknown"
+    MJ6 = "M4"
+    M3 = "M5"
```

Comment:
Should: Are these meant to be `MJ6` and `M3` or are they meant to be `M4` and `M5`?